scripts/globals/monstertpmoves.lua

```diff
@@ -396,7 +400,12 @@ function mobAddBonuses(caster, spell, target, dmg, ele)
 
     dmg = math.floor(dmg * burst);
 
-    mab = (100 + caster:getMod(MOD_MATT)) / (100 + target:getMod(MOD_MDEF)) ;
+    local mdefBarBonus = 0;
+    if (ele > 0 and ele <= 6 and target:hasStatusEffect(barSpells[ele])) then -- bar- spell magic defense bonus, held in subid
+        mdefBarBonus = target:getStatusEffect(barSpells[ele]):getSubType();
+		print("test")
```

Comment:
Whoops I forgot to remove that. As for bar spells: https://www.bg-wiki.com/bg/Category:Barspell#Increasing_Elemental_Barspell_Potency